free-1.md

```diff
@@ -57,12 +72,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This language would most likely be an internal DSL in Prolog, so most of the time, (maybe 80 or 90 percent) would be spent making language design decisions such as what criteria the user will be able to specify for optimized team divisions.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This feels like an easily extensible project. The base of the language should only take a few weeks to create and then the rest of the time would be spent adding additional features to make it work better and to allow the user to specify more criteria.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This woulc be a good idea for a project because it is easily scalable, it would address a real need for people at Mudd, and it works in a domain like Prolog where quite a bit less has been done by mainstream programmers because most programmers are more comfortable with languages like Python, Java, C or C++ which are signifcantly more difficult to use to solve these types of problems
+
```

Comment:
Out of curiosity, are you only intending this for CS clinic or do you intend this to be used for all Clinic disciplines?